index.js

```diff
@@ -386,13 +415,26 @@ class KeyringController extends EventEmitter {
       })
   }
 
-  // returns an app key
+  /**
+   * Gets the app key address for the given Ethereum address and origin.
+   *
+   * @param {string} _address - The Ethereum address for the app key.
+   * @param {string} origin - The origin for the app key.
+   * @returns {string} The app key address.
+   */
   async getAppKeyAddress (_address, origin) {
     const address = normalizeAddress(_address)
     const keyring = await this.getKeyringForAccount(address)
     return keyring.getAppKeyAddress(address, origin)
   }
 
+  /**
+   * TODO: what exactly does this do?
+   *
+   * @param {string} _address - The Ethereum address for the app key.
+   * @param {string} origin - The origin for the app key.
+   * @returns {string} The app key address. TODO: wut?
+   */
```

Comment:
Not totally sure what's going on here. Can someone please add a suggestion?

Reply:
It looks like this exports a an app key (e.g. the private key portion of it)

See here: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1775.md#private-app-key-generation-algorithm-1

Reply:
Got it, got it. Fixed.